bot/cogs/bot.py

```diff
@@ -1,10 +1,13 @@
 # coding=utf-8
-from discord import Embed
+import ast
+import time
+
+from discord import Embed, Message
 from discord.ext.commands import AutoShardedBot, Context, command, group
 
 from dulwich.repo import Repo
 
-from bot.constants import PYTHON_GUILD, VERIFIED_ROLE
+from bot.constants import HELP1_CHANNEL, HELP2_CHANNEL, HELP3_CHANNEL, PYTHON_CHANNEL, PYTHON_GUILD, VERIFIED_ROLE
```

Comment:
You know what to do.

Reply:
Either have two imports on each line, or have three on each line. It'll make things look nicer, promise!

Reply:
hmmmmmmmmmmmmmmmmmmmmmmmmm

Reply:
Can we have an equal number of imports spread out over either `2` or `3` lines.
Currently, the line looks too long for use in a production setting.